ui/reviewer/js/map.js

```diff
@@ -0,0 +1,49 @@
+mapboxgl.accessToken = 'pk.eyJ1IjoieWVzZW5pYW8iLCJhIjoiY2tlZjAyM3p5MDNnMjJycW85bmpjenFkOCJ9.TDYe7XRNP8CnAto0kLA5zA';
+
+const map = new mapboxgl.Map({
+    container: 'map', // container ID
+    // Choose from Mapbox's core styles, or make your own style with Mapbox Studio
+    style: 'mapbox://styles/mapbox/light-v9', // style URL
+    center: [-75.116728, 39.993436], // starting position [lng, lat],
+    zoom: 13, // starting zoom
+});
+
+// window.map = map;
+
+map.on('load', () => {
+     map.addSource('tileSet', {
+         type: 'vector',
+         'tileSize': 512,
+         scheme: "xyz",
+         tiles: ['https://storage.googleapis.com/musa509s23_team01_public/tiles/properties/{z}/{x}/{y}.pbf']
+     });
+
+     map.addLayer({
+         'id': 'my-layer',
+         'type': 'fill',
+         'source': 'tileSet',
+         'source-layer': 'property_tile_info',
+         'paint': {
+             // Use a match expression to create a property value-dependent style
+             'fill-color': [
+                'case',
+                ['all', ['>=', ['get', 'current_assessed_value'], 0], ['<=', ['get', 'current_assessed_value'], 100000]],
```

Comment:
**suggestion:** It may be useful to leave a comment in the code explaining what each of the colors means on the map, because the mapbox-gl style definition syntax isn't immediately  obvious to me. If I'm not mistaken, properties over $100,000 are blue, and under $100,000 are red currently?